sapi/cli/php_cli_server.c

```diff
@@ -2441,6 +2441,14 @@ static int php_cli_server_ctor(php_cli_server *server, const char *addr, const c
 		retval = FAILURE;
 		goto out;
 	}
+	// server_sock needs to be non-blocking when using multiple processes. Without it, the first process would
+	// successfully accept the connection but the others would block, causing client sockets of the same select
+	// call not to be handled.
+	if (SUCCESS != php_set_sock_blocking(server_sock, 0)) {
+		php_cli_server_logf(PHP_CLI_SERVER_LOG_ERROR, "Failed to make server socket non-blocking");
```

Comment:
I would suggest based on php_cli_server_log_level

Reply:
apart of this little remark, looks good to me.

Reply:
`php_cli_server_logf` contains a `if (php_cli_server_log_level < type) { return; }`, so this should be unnecessary.

Reply:
Ah right I missed that